cypress/integration/prospects/note.js

```diff
@@ -0,0 +1,39 @@
+describe('Create a prospect note', () => {
+  it('Should be able to create a prospect note', () => {
+    cy.login('normal@example.com', 'password')
+    cy.visit('http://localhost:3000/prospects/list')
+
+    // open the slide pane to create a prospect
+    cy.get('button[id="create-prospect"').click()
+
+    // fill the text fields for email, name, and phone
+    cy.get('input[id="mail"]').click().type('note@wellcode.com')
+    cy.get('input[id="name"]').click().type('prospect note')
+    cy.get('input[id="phone"]').click().type('0747070781')
+
+    // create a new prospect and check for success message
+    cy.get('button[id="create"').click()
+    cy.contains('Prospect successfully created!')
+
+    // open the first prospect
+    cy.visit('http://localhost:3000/prospects/profile/1')
```

Comment:
Here you should "click" the prospect from the list and make sure you click the prospect you just created

Reply:
I have completely removed this line because after the successful creation of this prospect, we will be automatically redirected to it and it will not require a "click".